src/Compilers/Core/Portable/Symbols/ITupleTypeSymbol.cs

```diff
@@ -14,7 +13,7 @@ namespace Microsoft.CodeAnalysis
     /// </remarks>
     public interface ITupleTypeSymbol : ITypeSymbol
```

Comment:
IDE would definitely like it.  Otherwise, it's going to be a PITA to work with a tuple.  If it's a named type instead, To find out the element types we'll have to look at .TypeArguments.  To find the tuple element names we'll have to look at all the fields.  etc. etc.  I personally like having this type.